### PR TITLE
Adding User Workload Monitoring Helm Chart

### PR DESCRIPTION
**Pull request by jordigilh**
Adds the User Workload Monitoring helm chart.

@sabre1041 @nasx please review at your discretion.

/Jordi

### CODE REVIEW

**Comment by sabre1041** [comment]
@jordigilh a couple comments

1. The templates currently are located in the `tests` template directory
2. Take a look at the inline comments

**Comment by jordigilh** [comment]
> @jordigilh a couple comments
> 
>     1. The templates currently are located in the `tests` template directory
> 
>     2. Take a look at the inline comments

1. Thanks, moved them out of `test` and into the parent directory.
2. Which comments are you referring to?

**Comment by sabre1041** [comment]
@jordigilh i didnt hit submit for the reviews. apologies. they should be visible now

File: helm/testing/user-workload-monitoring/user-workload-monitoring.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  targetRevision: helm-migration
+  project: default
+  server: https://kubernetes.default.svc
+  prune: true
+  repoURL: https://github.com/redhat-cop/rhel-edge-automation-arch.git
+  selfHeal: true
+
+charts:
+  user-workload-monitoring:
+    syncOptions:
```
[the remark this answers is not in the document]
Comment (jordigilh) [reply helm/testing/user-workload-monitoring/user-workload-monitoring.yaml:13]: By default it takes it from the common chart:
```
metadata:
  name: user-workload-monitoring
  namespace: openshift-gitops
```
Should I then explicitly target it to the `openshift-gitops` namespace?

File: charts/user-workload-monitoring/templates/tests/user-workload-monitoring-config.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.user-workload-monitoring-config.name }}
+  namespace: {{ .Values.user-workload-monitoring-config.namespace }}
```
[the remark this answers is not in the document]
Comment (jordigilh) [reply charts/user-workload-monitoring/templates/tests/user-workload-monitoring-config.yaml]: Sure, I'll change this.

[the remark this answers is not in the document]
Comment (jordigilh) [reply charts/user-workload-monitoring/templates/tests/user-workload-monitoring-config.yaml]: Done.

File: helm/testing/user-workload-monitoring/user-workload-monitoring.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jordigilh) [reply helm/testing/user-workload-monitoring/user-workload-monitoring.yaml:14]: changed to  ` {{ template "common.names.namespace" $ }}` in the chart template and defined as `destinationNamespace: openshift-gitops` in the yaml file inside `helm/testing`

**Comment by sabre1041** [comment]
the `destinationNamespace` is where the chart itself would be installed which in this case should be `openshift-monitoring`.  Also, I believe that both configmaps get configured in `openshift-monitoring`. if so, we can replicate the same configuration change so that it uses the namespace of the chart.

**Review by sabre1041 — APPROVED** [review]
LGTM

**Pull request merged**